sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -2446,6 +2455,7 @@ public void analyzePiiEntityRecognitionWithCategoriesFilters(HttpClient httpClie
                         IterableStream.of(asList(getExpectedRecognizePiiEntitiesActionResult(false, null,
                             TIME_NOW, getExpectedBatchPiiEntitiesForCategoriesFilter(), null))),
                         IterableStream.of(Collections.emptyList()),
+                        IterableStream.of(Collections.emptyList()),
```

Comment:
Sorry my Java knowledge is lacking so please educate me 😄, why do we need to call this so many times in a row?

Reply:
Most action tests share the same test method. So if it tests one specific action, we need to pass an empty list to other action results.